app/assets/javascripts/categories.js

```diff
@@ -0,0 +1,13 @@
+pages.categories.mobileCategories = function(){
+  $('.categories-select').on('change', function(){
+    var category = $(this).find('option:selected').text().toLowerCase();
+    var url = '/categories/' +
+      $(this).val() +
+      '-' +
```

Comment:
@zalapeach why are you putting this string into a new separated line?,  I think there is enough space in the previous line to be there.